sycl/test/basic_tests/queue/queue_parallel_for_interface.cpp

```diff
@@ -0,0 +1,81 @@
+// UNSUPPORTED: cuda
+// CUDA does not support unnamed lambdas.
```

Comment:
Can we avoid using "unnamed lambdas" to enable CUDA target?

Reply:
Yes. Done